torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -664,7 +667,7 @@ std::vector<ExprHandle> TensorExprKernel::sizesForValue(
 
   if (v->type()->isSubtypeOf(FloatType::get()) ||
       v->type()->isSubtypeOf(IntType::get())) {
-    return {1};
+    return {int64_t{1}};
```

Comment:
Is it the same as 1LL?

Reply:
`1ll` is probably fine although I'm not 100% sure.  There are a bunch of weirdnesses around implicit conversions, and the fact that on windows "long" and "long long" have different widths.  `int64_t` seemed like a good way to sidestep all that :-p